lib/util/commont.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { detectPlatform } from './common';
```

Comment:
Wrong filename I think?

Reply:
Why? I just used a new file for the function, as no existing matches because the function needs url and host-rules.
I also thought we find some more functions to put there. Didn't found a better name yet.

Reply:
Did you catch that Rhyse is probably referring to the name of the spec filename, that seems to contain an extra 't' after 'common'?

Reply:
🙈 didn't saw that. pushing a fix now